libraries/standard/http/include/http_client.h

```diff
@@ -319,7 +384,7 @@ typedef struct HTTPRequestInfo
  * @brief Callback to intercept headers during the first parse through of the
  * response as it is received from the network.
  */
-typedef struct httpResponseParsingCallback
+typedef struct HTTPClient_HeaderParsingCallback
```

Comment:
Seems like the **response** word was removed. How about `HTTPClient_**ResponseHeaders**ParsingCallback`

Reply:
I am good with adding "Response"